aten/src/ATen/cpu/vec/vec512/vec512_qint.h

```diff
@@ -0,0 +1,1195 @@
+#pragma once
+
+// DO NOT DEFINE STATIC DATA IN THIS HEADER!
+// See Note [Do not compile initializers with AVX]
+
+#include <ATen/cpu/vec/vec512/intrinsics.h>
+#include <ATen/cpu/vec/vec512/vec512_base.h>
+#include <ATen/native/quantized/affine_quantizer_base.h>
+#include <c10/util/qint32.h>
+#include <c10/util/qint8.h>
+#include <c10/util/quint8.h>
+
+#include <array>

```

Comment:
There is also a lot in this file that is straight up copy pasted without any modifications

Reply:
That's because I modified the `vec256` files to add AVX512 intrinsics.